### DISCUSSION TITLE
Addition to Durability Loss Ping

### DISCUSSION THREAD

**Discussion by FestusDM** [Zrips/CMI#4608] [Feature Requests]
Adding shears to the durability loss ping would be a good idea, as it is some what of a tool and players use it often.

---
Just like pickaxe, shears are something players use often. It'll be useful to the players to be able to know when their shears will break, just like other tools.

**Comment by CloudeLecaw** [comment]
You can give a shear the undestrucable enchant, then they dont loose dura anymore..
Simply hold item in Hand (or other player) and use cmi unbreakable (playerName) (true/false)

**Comment by Zrips** [comment]
Shears are already in tool category and should appear in action bar when its durability goes down under defined level.

**Comment by FestusDM** [comment]
> You can give a shear the undestrucable enchant, then they dont loose dura anymore..
> Simply hold item in Hand (or other player) and use cmi unbreakable (playerName) (true/false)

That's not the feature I'm looking for. Shear should still be destructable. Just looking for notification.

**Comment by FestusDM** [comment]
> Shears are already in tool category and should appear in action bar when its durability goes down under defined level.

Ah, I see it now.
Additional feature request according to this, then. The ability to change the % depending on which tool.

Example:
-Shears 30%
-Axe 10%
-Pickaxe 15%
